makefiles/suit.inc.mk

```diff
@@ -9,14 +9,19 @@ SUIT_COAP_SERVER ?= localhost
 SUIT_COAP_ROOT ?= coap://$(SUIT_COAP_SERVER)/$(SUIT_COAP_BASEPATH)
 SUIT_COAP_FSROOT ?= $(RIOTBASE)/coaproot
 
+BINDIR_SUIT = $(BINDIR)/suit_files
+$(BINDIR_SUIT): $(CLEAN)
+	$(Q)mkdir -p $(BINDIR_SUIT)
+
 #
-SUIT_MANIFEST ?= $(BINDIR_APP)-riot.suit.$(APP_VER).bin
-SUIT_MANIFEST_LATEST ?= $(BINDIR_APP)-riot.suit.latest.bin
-SUIT_MANIFEST_SIGNED ?= $(BINDIR_APP)-riot.suit_signed.$(APP_VER).bin
-SUIT_MANIFEST_SIGNED_LATEST ?= $(BINDIR_APP)-riot.suit_signed.latest.bin
+SUIT_MANIFEST_BASENAME ?= riot.suit
```

Comment:
getting the application name back in here should fix the overwriting for different applications?

Reply:
Instead, I added the application name to `SUIT_COAP_BASEPATH`, that way they can override each other while still keeping simple consistent file names.